components/02-elements/cookie-message/_cookie-message.scss

```diff
@@ -1,137 +1,82 @@
-$cookie-message__color      : $gray-lightest !default;
-$cookie-message__font-size  : $font-size-extra-small !default;
-$cookie-message__transition : height 0.3s !default;
-$cookie-message__close-width: 35px !default;
-$cookie-message__link-border: 1px solid $white !default;
-$cookie-message__icon-fill  : $white !default;
-$cookie-message__icon-width : 10px !default;
+$cookie-message__width               : 100% !default;
+$cookie-message__background-color    : $gray-light !default;
+$cookie-message__font-size           : $font-size-small !default;
+$cookie-message__font-family         : $font-family-base !default;
+$cookie-message__text-color          : $gray !default;
+$cookie-message__text-padding        : $spacer--medium $spacer--medium 0 $spacer--medium !default;
+$cookie-message__text-padding--small : $spacer--medium $spacer $spacer--medium $spacer--medium !default;
+$cookie-message__close-size          : 48px !default;
+$cookie-message__close-font-weight   : $font-weight-bold !default;
+$cookie-message__close-padding       : 0 $spacer--medium 0 0;
+$cookie-message__close-padding--small: 0 $spacer--medium 0 $spacer default;
+$cookie-message__close-icon-fill     : $color-primary !default;
+$cookie-message__close-icon-size     : 12px !default;
 
 .cookie-message {
+    display: none;
+    align-items: stretch;
     width: $cookie-message__width;
-    background-color: $cookie-message__color;
-    transition: $cookie-message__transition;
+    background-color: $cookie-message__background-color;
     box-sizing: border-box;
+    font-size: $cookie-message__font-size;
+    font-family: $cookie-message__font-family;
 
     &__container {
-        position: relative;
-        display: block;
-        margin: 0 auto;
-        max-width: $max-content-width;
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: flex-end;
+        align-items: center;
+        min-width: 100%;
         text-align: center;
-        padding: $cookie-message__padding 35px;
-    }
-
-    &__message {
-        display: block;
-        margin: 0;
-        text-align: center;
-    }
-
-    &__text {
-        font-size: $cookie-message__font-size;
-        font-weight: 700;
-        display: block;
-        margin-bottom: $cookie-message__margin;
-        text-transform: uppercase;
-
-        @include mq($screen-m) {
-            display: inline-block;
-            margin-bottom: 0;
-            margin-right: 35px;
+        @include mq($screen-s) {
+            flex-wrap: nowrap;
+            justify-content: space-between;
         }
     }
 
-    &__link {
-        @include links() {
-            display: inline-block;
-            font-weight: 400;
-            font-size: $cookie-message__font-size;
-            border: $cookie-message__link-border;
-            padding: 5px;
+    &__text {
+        flex-basis: 100%;
+        color: $cookie-message__text-color;
+        text-align: justify;
+        padding: $cookie-message__text-padding;
```

Comment:
Why not use padding on container to top/bottom and keep here only for right/left? And then better to use margin instead of padding for right/left values.

Reply:
Using container padding was making to much space on unwrapped text, and too little on wrapped. The rendered content looks better this way. About the text's right padding, its not linked with clickable area so I'll leave it this way.

Reply:
ok no problem